ruvnl_consumer_app/app.py

```diff
@@ -82,7 +83,11 @@ def fetch_data(data_url: str) -> pd.DataFrame:
 
             start_utc = dt.datetime.fromtimestamp(int(record["SourceTimeSec"]), tz=dt.UTC)
             power_kw = record["Average2"] * 1000  # source is in MW, convert to kW
-
+            if v=="wind":
+                if(start_utc<dt.datetime.now(dt.timezone.utc)-dt.timedelta(hours=0)):
+                        timestamp_after_raise = f"Timestamp: {(dt.datetime.now(pytz.timezone('Asia/Calcutta')))}"
+                        raise Exception("Start time is at least 1 hour old. " + timestamp_after_raise )
```

Comment:
It would be great to include {start_utc}, so it could be 
`f"Start time is at least 1 hour old. {timestamp_after_raise} {start_utc=}"

Reply:
`start_ist = start_utc. astimezone(pytz.timezone('Asia/Calcutta'))`